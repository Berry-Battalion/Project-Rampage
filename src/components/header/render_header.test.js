import React from 'react';
import Header from './header';
import renderer from 'react-test-renderer';

describe('<Header />', () => {
    test('renders correctly', () => {
        const tree = renderer.create(
            <Header />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
})