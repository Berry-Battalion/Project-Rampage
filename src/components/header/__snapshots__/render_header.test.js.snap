// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<nav
  className="sc-bdVaJa pEDwD"
>
  Header
</nav>
`;
